### PR TITLE
Refactor migration guide

### PR DESCRIPTION
**Pull request by ayshsandu**

### CODE REVIEW

File: en/docs/administer/browsing-the-h2-database.md [hunk 1]
```diff
@@ -62,7 +58,7 @@ browse through it.
       </tr>
       <tr class="even">
       <td><strong>JDBC URL</strong></td>
-      <td>location of the H2 database</td>
+      <td>location of the H2 database. All the default databases are located at `<IS_HOME>/repository/database` location.</IS_HOME></td>
       </tr>
```
Comment (sherenewso2) [inline en/docs/administer/browsing-the-h2-database.md]: This should be "All the default databases are located at `<IS_HOME>/repository/database`." We can remove the "location.<IS_HOME>" part.

File: en/docs/setup/migrating-custom-scope-validators.md [hunk 2]
```diff
@@ -81,10 +82,10 @@ validators to IS 5.5.0, follow the instructions below:
 
 3.  If you want to use the XACML based scope validator introduced in
     WSO2 IS 5.5.0 together with your custom scope validators, add the
-    following line under `           <ScopeValidators>          ` in the
-    `           <IS_HOME>/repository/conf/identity/identity.xml          `
-    file:
+    following configuration to the ` deployment.toml ` file in the `
+    <IS_HOME>/repository/conf/ ` file:
```
Comment (sherenewso2) [inline en/docs/setup/migrating-custom-scope-validators.md]: shall we change this to "...conf/ **folder** "

File: en/docs/setup/migrating-data-publishers.md [hunk 3]
```diff
@@ -94,20 +85,16 @@ instead.
         }
     ```
 
-3.  To enable the new event handler, add handler and relevant property to the
-    `default.json` file found in the`<IS_HOME>/repository/resources/conf` folder.
-    For example, following properties were added to define the
+3.  To enable the new event handler, add handler and relevant property
+    to the `deployment.toml` file found in
+    the`<NEW_IS_HOME>/repository/conf` folder. For example, following
```
Comment (sherenewso2) [inline en/docs/setup/migrating-data-publishers.md]: missing space between "the" and "`<NEW_IS_HOME>"

File: en/docs/setup/migrating-to-590.md [hunk 4]
```diff
@@ -1,22 +1,15 @@
 # Migrating to 5.9.0
 
-!!! note
-    Before you follow this section, see [Before you begin](../../setup/migrating-before-you-begin) and [Preparing for 
-    migration](../../setup/migrating-preparing-for-migration) sections to read on prerequisites.
+Before you follow this section, see
+[Before you begin](../../setup/migrating-before-you-begin) and
+[Preparing for migration](../../setup/migrating-preparing-for-migration)
+sections to read on prerequisites.
```
Comment (sherenewso2) [inline en/docs/setup/migrating-to-590.md]: Shall we change this to one of these:

"to read **the** prerequisites" or "read **up on** prerequisites" or "read **about** prerequisites"

File: en/docs/setup/migrating-to-590.md [hunk 5]
```diff
@@ -90,9 +83,13 @@ Follow the steps below to perform the upgrade.
     `          <NEW_IS_HOME>/repository/components/dropins         `
     directory.
     
-    !!! note
+    !!! warning "Important" 
         You may need to update the custom components to work with WSO2 Identity Server 5.9.0, 
         refer [Migrating custom components](../../setup/migrating-preparing-for-migration/#migrating-custom-components).
+        If applicable migrate
```
Comment (sherenewso2) [inline en/docs/setup/migrating-to-590.md]: If applicable **,** migrate [Data Publishers](../../setup/migrating-data-publishers) and [Custom Scope validators](../../setup/migrating-custom-scope-validators) **by** referring to the relevant document.

File: en/docs/setup/migrating-to-590.md [hunk 6]
```diff
@@ -118,14 +115,17 @@ Follow the steps below to perform the upgrade.
...
     
-    !!! warning
-        If you are using a version prior to WSO2 Identity Server 5.8.0, then proceed to the steps 6 to 9 
-        otherwise directly proceed to step 9.
-    
-6.  Do the following database updates:  
-    1.  Download the [migration resources](../../assets/attachments/migration/wso2is-5.9.0-migration.zip)
+6. Make sure that you have migrated the configurations into NEW version
+   as advised in
+   [preparing for migration section.](../../setup/migrating-preparing-for-migration/#migrating-the-configurations)
```
Comment (sherenewso2) [inline en/docs/setup/migrating-to-590.md]: Ensure that you have migrated the configurations over to the NEW version as instructed in the ....

File: en/docs/setup/migrating-to-590.md [hunk 7]
```diff
@@ -169,266 +169,268 @@ Follow the steps below to perform the upgrade.
...
-    
-    CREATE TABLE IDN_AUTH_SESSION_APP_INFO (
-              SESSION_ID VARCHAR (100) NOT NULL,
-              SUBJECT VARCHAR (100) NOT NULL,
-              APP_ID INTEGER NOT NULL,
-              INBOUND_AUTH_TYPE VARCHAR (255) NOT NULL,
-            PRIMARY KEY (SESSION_ID, SUBJECT, APP_ID, INBOUND_AUTH_TYPE)
-    )
-    /
+10.  Stop the server once the task of migration client is completed.
```
Comment (sherenewso2) [inline en/docs/setup/migrating-to-590.md]: Shall we change this to "Stop the server once the migration client task has completed"

File: en/docs/setup/migrating-to-590.md [hunk 8]
```diff
@@ -169,266 +169,268 @@ Follow the steps below to perform the upgrade.
...
+
+   
+    !!! Tip 
+        If you are migrating from WSO2 Identity Server 5.8.0, then you can skip the steps 8-10 and run following the following DB script against the IDENTITY_DB.
+
```
Comment (sherenewso2) [inline en/docs/setup/migrating-to-590.md]: Remove the word "the" in "skip **the** steps"

File: en/docs/administer/browsing-the-h2-database.md (hunk 1 above)
Comment (ayshsandu) [reply en/docs/administer/browsing-the-h2-database.md]: fixed

File: en/docs/setup/migrating-to-590.md (hunk 5 above)
Comment (ayshsandu) [reply en/docs/setup/migrating-to-590.md]: fixed

File: en/docs/setup/migrating-to-590.md (hunk 6 above)
Comment (ayshsandu) [reply en/docs/setup/migrating-to-590.md]: fixed

File: en/docs/setup/migrating-to-590.md (hunk 7 above)
Comment (ayshsandu) [reply en/docs/setup/migrating-to-590.md]: fixed

File: en/docs/setup/migrating-to-590.md (hunk 8 above)
Comment (ayshsandu) [reply en/docs/setup/migrating-to-590.md]: fixed

**Pull request merged**